### PR TITLE
new authorizer approach

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-api/terraform/bin/deploy-app.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ fi
 
 # build the cognito authorizer using parcel
 pushd ../template/cognito-authorizer
-npx parcel build index.js --target node --bundle-node-modules
+npx parcel build index.js --target node --bundle-node-modules --no-minify
```
Comment (kkoskelin) [inline web-api/terraform/bin/deploy-app.sh:26]: sorry for the post-merge reviews.  can you tell me more about why `--no-minify`? Is this related to HB, or something else?
